Configure

```diff
@@ -1061,7 +1061,6 @@ while (@argvcopy)
                         push @{$useradd{LDFLAGS}}, $_;
                         }
                 elsif (/^-rpath$/ or /^-R$/)
```

Comment:
```suggestion
                elsif (/^-R$/)
```
Or is the -rpath still needed for other reasons? If so it should be documented why.

Reply:
I don't know if some other system needs it and the comment was just never updated.  I figure that is more than likely.

Reply:
Oh, it's the -rpath flag that lld also has. Never mind my suggestion then (I should have looked at the context).